integral_view/forms/zfs_forms.py

```diff
@@ -141,7 +141,20 @@ def __init__(self, *args, **kwargs):
 
 
 class ImportPoolForm(forms.Form):
-    name = forms.CharField()
+    def __init__(self, *args, **kwargs):
+        pools = []
+        if kwargs and ('exported_pools' and 'destroyed_pools' in kwargs):
```

Comment:
Could there be a case where there is only either one of the exported_pools or the destroyed_pools?

Reply:
Well, it's okay if the lists(arg type) are empty but both the key word arguments​ must be passed. Mandating it ensures that one or the other is not missed inadvertently.